migrations/1665416411399_allow-profile-picture-field-when-creating-a-user.js

```diff
@@ -0,0 +1,7 @@
+exports.shorthands = undefined
+
+exports.up = pgm => {
+	pgm.addColumns('users', { profilePictureUrl: { type: 'text'}} )
```

Comment:
@beblicarl Please 'text' instead of varchar ?

Reply:
@jocrah I am done. Although, I was wondering if we should also include the field in the create user tests? @beblicarl

Reply:
okay thanks @itzgeebee. 

>  I was wondering if we should also include the field in the create user tests

yea makes sense

Reply:
> @beblicarl Please 'text' instead of varchar ?

Both work so any goes and to maintain consistency text was used throughout when creating an image field

Reply:
> okay thanks @itzgeebee.
> 
> > I was wondering if we should also include the field in the create user tests
> 
> yea makes sense

If all other test cases ensure that things are right, what difference will one additional one make?

Reply:
> > @beblicarl Please 'text' instead of varchar ?
> 
> Both work so any goes and to maintain consistency text was used throughout when creating an image field

Okay then. No issues from me.

Reply:
> If all other test cases ensure that things are right, what difference will one additional one make?

@beblicarl :). So the logic is that, if i come to your code and i take out your feature, the tests should be able to give a signal. In this case, the tests are passing with or without the code in this pr so easy to break and will go unnoticed till an actual enduser experiences the bug in production

The tests are needed to answer the following questions (and more if any):

1. is the profilePictureUrl is really being entered when the user adds it?
2. is it being returned as part of the response now
3. is the request only accepting a valid url for the profilePictureUrl field and no other kind of input

Reply:
> > okay thanks @itzgeebee.
> > > I was wondering if we should also include the field in the create user tests
> > 
> > 
> > yea makes sense
> 
> If all other test cases ensure that things are right, what difference will one additional one make?

@beblicarl I was just thinking about coverage. Since we have the field, we might as well test if wrong data is entered